### ISSUE TITLE
allow fine tuning when cues play within a line

### ISSUE THREAD

**Issue by hediet** [microsoft/vscode#188285]
Diff Editor: Don't play audio cue when navigating in a line

**Comment by meganrogge** [comment]
we should have a setting for this because some users want the cues to play while navigating in the line

**Comment by meganrogge** [comment]
/duplicate [#204257](https://github.com/microsoft/vscode/issues/204257)
